core/random.rb

```diff
@@ -182,3 +182,5 @@ def bytes(length)
     s
   end
 end
+
+Random::DEFAULT = Random.new
```

Comment:
Oh, that's a smart trick. I was looking for directories like `delta` but I guess we're no longer doing that.

Completely off topic: May I ask if `zed.rb` means anything? If it's just lexically sorted the last, I would imagine using something like `zzz.rb` would be funny.